server/src/libs/environment/types.ts

```diff
@@ -0,0 +1,37 @@
+export interface EnvConfig {
+  [name: string]: string | undefined
+}
+
+export interface Environment {
+  __DEV__: boolean
+  ENV: string
+  REGION:
+    | 'us-central1'
+    | 'us-east1'
+    | 'us-east4'
+    | 'europe-west1'
+    | 'europe-west2'
+    | 'asia-east1'
+    | 'asia-east2'
+    | 'asia-northeast1'
+    | 'asia-northeast2'
+    | 'us-west2'
+    | 'us-west3'
+    | 'us-west4'
+    | 'northamerica-northeast1'
+    | 'southamerica-east1'
+    | 'europe-west3'
+    | 'europe-west6'
+    | 'europe-central2'
+    | 'australia-southeast1'
+    | 'asia-south1'
+    | 'asia-southeast1'
+    | 'asia-southeast2'
+    | 'asia-northeast3'
+  APP_PUBLIC_URL: string
+  APP_PROXY_URL: string
+  API_BASE_URL: string
```

Comment:
à l'inverse, ici, on pourrait etre plus précis

```suggestion
  APP_PUBLIC_URL: URL
  APP_PROXY_URL: URL
  API_BASE_URL: URL
```

Reply:
Je ne sais pas si ceci est correct, puisque le fichier contiens des string, il faudrait les parse en URL.

Reply:
en effet, il faudrait les parser en URL, ce qui ajouterait une garantie supplémentaire